conjure-java-core/src/main/java/com/palantir/conjure/java/util/Javadoc.java

```diff
@@ -82,7 +82,7 @@ public static Optional<String> getParameterJavadoc(ArgumentDefinition argument,
                         + JavaNameSanitizer.sanitizeParameterName(
                                 argument.getArgName().get(), endpoint)
                         + " "
-                        + Javadoc.render(argument.getDocs().get()));
+                        + Javadoc.render(docs));
```

Comment:
ha, good catch.

(no change requested, up to you if you'd like to break it out) For this sort of drive-by on a larger PR, sometimes I prefer to create a new small PR into develop for the cleanup. That way it's super easy to review, and keeps the large feature PRs cleaner.

Reply:
Made #2406 instead of adding the drive-by here.